.github/workflows/build.yaml

```diff
@@ -48,7 +49,13 @@ jobs:
       - name: Install dependencies
         run: mvn clean install -DskipTests=true -Dmaven.javadoc.skip=true --batch-mode --show-version --file pom.xml
       - name: Build with Maven
-        run: mvn clean install && mvn test -Pskip-default-test -Pintegration-test --batch-mode --file pom.xml
+        run: |
```

Comment:
Shouldn't the activation rule in the POM make this unnecessary?

Reply:
This was necessary because I was using an explicit activation rule. I updated the pom with an implicit activation rule and removed this.